<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessGameClass</class>
 <widget class="QMainWindow" name="ChessGameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChessGame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="GameWidget" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>351</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="ChessBoard" rowstretch="0" columnstretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QLabel" name="NotificationLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>440</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="MoveRecord">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>141</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BackToMenuButton">
     <property name="geometry">
      <rect>
       <x>444</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#BackToMenuButton{
border-style: outset;
border-width: 2px;
border-color: #F39C12;
border-radius: 11px;
}</string>
     </property>
     <property name="text">
      <string> To main menu </string>
     </property>
    </widget>
    <widget class="QPushButton" name="NewGameButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#NewGameButton{
padding: 4px;
border-style: outset;
border-width: 2px;
border-color: #F39C12;
border-radius: 11px;
}</string>
     </property>
     <property name="text">
      <string>New Game</string>
     </property>
    </widget>
    <widget class="QWidget" name="ComfirmNewGameWidget" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QDialogButtonBox" name="ComfirmNewGameButtons">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>156</width>
        <height>23</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Start new game?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ChangePawnWidget" native="true">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>60</y>
       <width>181</width>
       <height>411</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>12</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose figure:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="ChangePawnGroupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>160</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="FigureList">
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Queen</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>Rook</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>Knight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Bishop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>Pawn</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="ChangePawnButton">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>210</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#ChengePawnButton{
padding: 4px;
border-style: outset;
border-width: 2px;
border-color: #F39C12;
border-radius: 11px;
}</string>
      </property>
      <property name="text">
       <string>Change my pawn!</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="MainMenu" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>581</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#MainMenu{
	background-color: white;
}</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>341</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QLabel" name="MenuLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>The best chess game ever!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StartGameButton">
        <property name="styleSheet">
         <string notr="true">QPushButton#StartGameButton{
padding: 6px;
    border-style: outset;
border-width: 3px;
border-color: green;
border-radius: 15px;
}</string>
        </property>
        <property name="text">
         <string>Start New Game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ExitButton">
        <property name="styleSheet">
         <string notr="true">QPushButton#ExitButton{
padding: 4px;
border-style: outset;
border-width: 2px;
border-color: red;
border-radius: 12px;
}</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ChessGame.qrc"/>
 </resources>
 <connections/>
</ui>
